ownership/ownership.go

```diff
@@ -12,112 +12,114 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package keysync
+package ownership
 
 import (
-	"bufio"
 	"fmt"
-	"os"
+	"os/user"
 	"strconv"
-	"strings"
 
 	"github.com/sirupsen/logrus"
 )
 
+type Lookup interface {
+	UID(username string) (uint32, error)
+	GID(groupname string) (uint32, error)
+}
+
+type Os struct {}
```

Comment:
It looks like `Os` is expected to always implement `Lookup`; is it worth adding a 
```
var _ Lookup = Os{}
```
to make sure?